docs/getting_started/contribution_guide.rst

```diff
@@ -59,4 +59,130 @@ Kiso-testing and make sure that you understand your rights and obligations.
 Any contributions to Kiso-testing project code base needs to be licensed under
 EPL v2.0.
 
+What should I do before committing ?
+------------------------------------
+
+PEP8-Compliancy
+---------------
+In order to maintain clear and user-friendly project, make sure that your
+changes respect PEP8 standards. PEP8 is a guide that provides Python coding
+conventions (naming, indentation,...).
+Official document : https://peps.python.org/pep-0008/
+
+To make sure your changes are PEP8 Compliant, different tools exist to help you
+here:
+    - linter (applicable on IDE) :
+        show some warning directly on IDE
+    - pre-commit hook :
+        hook scripts that lint the added code using
+    flake8 and format it using black and isort (for
+    installation see :ref:`getting_started`)
```

Comment:
I think the reference is missing in getting_started.rst (`.. _getting_started`)

Reply:
It does not work neither with one or the other implemetation, just will replace with no reference